src/frontend/src/catalog/system_catalog/mod.rs

```diff
@@ -404,6 +404,7 @@ prepare_sys_catalog! {
     { BuiltinCatalog::Table(&RW_HUMMOCK_BRANCHED_OBJECTS), read_hummock_branched_objects await },
     { BuiltinCatalog::Table(&RW_HUMMOCK_COMPACTION_GROUP_CONFIGS), read_hummock_compaction_group_configs await },
     { BuiltinCatalog::Table(&RW_HUMMOCK_META_CONFIGS), read_hummock_meta_configs await},
+    { BuiltinCatalog::Table(&RW_HUMMOCK_COMPACT_TASK_ASSIGNMEN), read_hummock_compaction_status await },
```

Comment:
typo: ASSIGNMEN

Since the result doesn't contain any information about the assignment of the compact task, how about calling it `RW_HUMMOCK_COMPACTION_STATUS`